x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/TransformFeatureSetUsage.java

```diff
@@ -17,25 +17,36 @@
 import org.elasticsearch.xpack.core.XPackField;
 
 import java.io.IOException;
+import java.util.Collections;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Objects;
 
 public class TransformFeatureSetUsage extends Usage {
 
+    private static final String FEATURE_COUNTS = "feature_counts";
+
     private final Map<String, Long> transformCountByState;
+    private final Map<String, Long> transformCountByFeature;
     private final TransformIndexerStats accumulatedStats;
 
     public TransformFeatureSetUsage(StreamInput in) throws IOException {
         super(in);
         this.transformCountByState = in.readMap(StreamInput::readString, StreamInput::readLong);
+        if (in.getVersion().onOrAfter(Version.CURRENT)) {  // TODO: V_7_13_0
```

Comment:
```suggestion
        if (in.getVersion().onOrAfter(Version.V_8_0_0)) {  // TODO: V_7_13_0
```
This is a minor thing (feel free to ignore this), but I find it useful to make it `V_8_0_0`, so that it won't even compile without changing it.

Reply:
Done.